pyproject.toml

```diff
@@ -74,6 +74,8 @@ requires = [
     "oauth2-lib==1.0.13; python_version >= '3.10'",
     "markupsafe==2.0.1",
     "bandit==1.7.2",
+    "redis==4.3.1",
```

Comment:
this isn't specifically needed here, the readme tells you to install it separate from core.